### PR TITLE
#1852189: buyer should be able to view cart

### PR DESCRIPTION
**Pull request by Gabriel-mdv**
## What does this PR do?
- Allow buyer to view cart anywhere in the app

## How should this be manually tested?

- Clone the repository using `git clone https://github.com/atlp-rwanda/e-comm-team-techsmith-fn`

- Checkout to the branch using `git checkout <branch-name>`

- Install node packages that the application depends on: `npm i`

- Create a `.env` file in the root directory and add all environment variables required to run the application. The required variables are listed in the `.env.example` file available in root directory of the application.

- Start the application development server: `npm start`

- Check out the components documentation: `npm run storybook`

## What are the relevant pivotal tracker/Trello stories?
[#1852189](https://www.pivotaltracker.com/story/show/185218934)

## Screenshots (if appropriate)

Screenshot 2023-06-16 at 12 32 41

Screenshot 2023-06-16 at 01 43 19
Screenshot 2023-06-16 at 01 39 29

### CODE REVIEW

File: .eslintrc.cjs [hunk 1]
```diff
@@ -12,21 +12,26 @@ module.exports = {
   plugins: ['react'],
   rules: {
     'spaced-comment': 'off',
+    'react/prop-types': 'off',
     'react/button-has-type': 'off',
-    "no-use-before-define": "off",
+    'no-use-before-define': 'off',
```
Comment (jcharles921) [inline .eslintrc.cjs:17]: I don't agree with you in removing the following rules :

- '**no-use-before-define': 'off'**
- **'import/no-named-as-default-member': 'off'**
- **'import/no-named-as-default': 'off'**
disabling them may lead to avoidable errors in our codes

File: src/assets/index.js [hunk 2]
```diff
@@ -13,3 +13,4 @@ export { default as lock } from './svg/Lock.svg';
 export { default as loading } from './svg/loading.svg';
 export { default as Left } from './svg/Left.svg';
 export { default as Right } from './svg/Right.svg';
+export { default as whiteshoes } from './whiteshoes.png';
```
Comment (jcharles921) [inline src/assets/index.js:16]: As our TTL requested it , we should put png in the images folder

**Review by jcharles921 — COMMENTED** [review]
Just some changes to apply nothing big  😅 😅

**Review by jcharles921 — CHANGES_REQUESTED** [review]
Just some changes to apply nothing big 😅 😅

File: vite.config.js.timestamp-1686515408485-7ded070124dfd.mjs
Comment (jcharles921) [inline vite.config.js.timestamp-1686515408485-7ded070124dfd.mjs:1]: This file should not be pushed

File: .eslintrc.cjs (hunk 1 above)
Comment (niyontwali) [reply .eslintrc.cjs:17]: Great insight @jcharles921

File: src/components/Cart.jsx [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+      )}
+    </div>
+  );
+};
+
+export default Cart;
+
+// user credentials with some things in carts
+// "email": "joshua1@gmail.com",
+// "password": "Testing@123",
```
Comment (niyontwali) [inline src/components/Cart.jsx:172]: Please remove all these unnecessary comments

**Review by niyontwali — CHANGES_REQUESTED** [review]

**Comment by niyontwali** [comment]
@Gabriel-mdv, kindly ensure that when the cart is cleared, the functionalities related to ordering are not visible. It logically follows that you should not be able to order items that are no longer present in the cart.
Screenshot 2023-06-17 at 11 24 17

**Comment by niyontwali** [comment]
Oooh another thing, @Gabriel-mdv please squash your commits to one.

**Review by irakozejules — APPROVED** [review]

File: src/components/AddRemoveButton.jsx
Comment (frankflej) [inline src/components/AddRemoveButton.jsx:1]: Can you remove the commented on click function if its unnecessary

**Review by frankflej — COMMENTED** [review]
So far your code looks good but you have many unnecessary comments which should be removed

File: src/containers/SignupContainer.jsx
Comment (irakevine) [inline src/containers/SignupContainer.jsx:1]: @Gabriel-mdv  you can also remove  empty line

File: src/styles/cart.scss
Comment (irakevine) [inline src/styles/cart.scss:1]: here agin you can remove unnecessary comments

**Pull request closed without merging**